services/mac_manager/src/main/java/com/futurewei/alcor/macmanager/controller/MacController.java

```diff
@@ -201,14 +202,14 @@ public MacRangeJson updateMacRange(@PathVariable String rangeid, @RequestBody Ma
     @RequestMapping(
             method = DELETE,
             value = {"/macs/ranges/{rangeid}", "/v4/macs/ranges/{rangeid}"})
-    public ResponseId deleteMacRange(@PathVariable String rangeid) throws Exception {
+    public String deleteMacRange(@PathVariable String rangeid) throws Exception {
```

Comment:
Hi Eunju, I'd like to say that it is better to use ResponseId here, because this return type is widely used in other managers.

Reply:
No problem. I changed it.